Description: replace mpg123 by a sort of mplayer
Author: Nicolas HOUDELOT <nicolas@demosdebs.org>
Applied-Upstream: no
Last-Update: 2020-05-01
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/main.cc
+++ b/main.cc
@@ -12,7 +12,7 @@
 *******************************************************************************/
 void musik() {
 	usleep(1500000); //ugly hack
-	system("mpg123 gnusong.mp3");
+	system("mplayer gnusong.mp3");
 }
 
 int main(int argc,char *argv[]) {
@@ -38,7 +38,3 @@
 	}
 }
 
-
-
-
-
